### ISSUE TITLE
[GitHub] TypeError: Cannot read properties of null (reading 'name')

### ISSUE THREAD

**Issue by martylouis** [raycast/extensions#4831]
### Extension

https://www.raycast.com/raycast/github

#### Error:
```
TypeError: Cannot read properties of null (reading 'name')

Iy:IssueActions.tsx:31:39
    at ray-list-item
xn:index.js:18:20475
YA:search-repositories.js:1370:1712
```

### Steps To Reproduce

1. Open Raycast
2. Type `github`
3. Choose any Open Issues or Created Issues
4. View Issues
5. _Issues load for less then a second and then "Something went wrong" screen_

### Current Behaviour

Screen recording: https://share.cleanshot.com/7YDZHM4C

**Comment by thomaslombart** [comment]
I couldn't reproduce the bug with my GitHub account. There must be something in your issues that isn't handled by the GitHub extension. Could you link one or two issues you tried loading so that I can see what specific detail we missed in the extension?

**Comment by martylouis** [comment]
@thomaslombart thanks for taking a look. 

So, I work mostly with private repos with private issues. But when I switched one private repo to public, to send you some issue links, I checked the plugin response, and I could now see all of my issues, including the private ones.

So, I added an issue to a public repo, and the plugin now works as expected. 

So, if you have all private repos with issues, but at least one public repo with one open issue, it works as expected. Not sure if that's a bug or not. 

![Arc--0223100445](https://user-images.githubusercontent.com/169188/220964317-ff317c1d-9235-426d-9ba0-ed815c477e3c.png)

So I guess I'll just always have something to work on. 😅
